lib/api/src/sys/mod.rs

```diff
@@ -42,7 +42,7 @@ pub use wasmer_derive::ValueType;
 pub use wasmer_types::is_wasm;
 pub use wasmer_types::{
     CpuFeature, ExportType, ExternType, FunctionType, GlobalType, ImportType, MemoryType,
-    Mutability, TableType, Target, Type,
+    Mutability, OnCalledAction, TableType, Target, Type,
```

Comment:
Can we add a new comment linking to the asyncify refactor afterwards:

```
// TODO: issue #XYZ 
```

Reply:
done